replication_report-extensions.Rmd

```diff
@@ -0,0 +1,320 @@
+pwr.chisq.test(w = .39, N = 91, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 91, df = 1, sig.level = 0.05, power = .8)
+
+# Analyses on SUBSETTED sample:
+
+# Filter out additional 19 Ss with too positive attitudes in "disliked music" condition & too negative attitudes in "liked music" condition:
+
+# Find Cronbach's Alpha for music attitudes scale (for comparison with article):
+#dat_music <- dat_e %>%
+#  select(att_muz1, att_muz2, att_muz3)
+#View(dat_music)
+#alpha(dat_music, na.rm = TRUE, delete=TRUE)
+
+# Find mean attitudes for each music condition (for comparison with article): 
+#dat_e %>%
+#  group_by(music) %>%
+#  summarise(avg = mean(c(att_muz1, att_muz2, att_muz3)))
+
+# Add mean attitude column to dataframe:
+dat_e <- dat_e %>%
+  mutate(avg_att_muz = (att_muz1 + att_muz2 + att_muz3) / 3)
+#View(dat_e)
+
+# Check those means (for comparison with article):
+#dat_e %>%
+#  group_by(music) %>%
+#  summarise(avg = mean(avg_att_muz))
+
+# Exclude when average attitude > 3 in disliked cond. or average attitude <3 for liked cond. (1 == liked music, 2 == disliked music)
+dat_egood <- dat_e %>%
+  filter(music == 1 & avg_att_muz > 3 | music == 2 & avg_att_muz < 3)
+#View(dat_egood)
+
+## Redo Chi-Square with subsetted data:
+
+# Test whether music condition is independent of pen choice:
+# Create 2x2 contingency table of frequencies - pen choice by music condition
+tbl2 <- table(dat_egood$music, dat_egood$chose_advertised_pen) 
+tbl2    
+
+# Chi-Square, no correction, no simulation:
+cs2 <- chisq.test(tbl2, correct = FALSE)
+tidy(cs2)
+
+# Power analysis -- Find the power to detect .35 effect size reported in paper:
+pwr.chisq.test(w = .35, N = 72, df = 1, sig.level = 0.05, power = NULL)
+
+# Power analysis -- Find the effect size for a test with minimum of .8 sensitivity:
+pwr.chisq.test(w = NULL, N = 72, df = 1, sig.level = 0.05, power = .8)
+
+# Confidence Intervals (not replicated from paper):
+#upper (95%)
+#CI95_up <- qchisq(.95, df = 1)
+#lower (5%)
+#CI95_low <- qchisq(.05, df = 1)
+
+## Replicate Table: Frequency table & chi-square statistic of subsetted data
+# (replicate "Experiment 3" portion of the table from article)
+
+# Generate count and proportion (percent) tallies for replicating the table:
+# Amongst those in Liked Music condition:
+dat_egoodLike <- dat_egood %>%
+  filter(music == 1)
+dat_egoodLike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodLike) * 100)
+
+# Amongst those in Disliked Music condition:
+dat_egoodDislike <- dat_egood %>%
+  filter(music == 2)
+dat_egoodDislike %>%
+  group_by(chose_advertised_pen) %>%
+  summarise(n = n(),
+            percent = n / nrow(dat_egoodDislike) * 100)
+
+# Generate table:
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
```

Comment:
I don't think vector names are supposed to be capitalized according to the style guide.